src/App.tsx

```diff
@@ -15,48 +16,117 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ALPHABETICALLY = 'alphabetically',
+  LENGTH = 'length',
+  NONE = '',
+}
+
+const getPreparedGoods = (
+  goods: string[],
+  sortField: string,
```

Comment:
It will be better to create new interface, with SortType and isReversed. 
Like this:
interface SortParams { 
sortType: SortType,
isReversed: boolean,
}

And in this function write:
{ sortType, isReversed }: SortParams

Reply:
agree with [comment](https://github.com/mate-academy/react_list-of-goods-hooks/pull/1112/files#r1259251451)